tools/win-update-e2e/window-watch.ps1

```
@@ -0,0 +1,91 @@
+  $now = (Get-Date).ToString('o')
+  foreach ($w in $windows) {
+    $handle = [long]$w.handle
+    if ($baselineHandles.Contains($handle)) { continue }
+
+    $title = [string]$w.title
+    $prior = $null
+    $isNew = -not $titleByHandle.ContainsKey($handle)
+    if (-not $isNew) { $prior = $titleByHandle[$handle] }
+    if ($isNew -or $prior -ne $title) {
+      $titleByHandle[$handle] = $title
+      $event = [pscustomobject]@{
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'event' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)